powershell/VstsTaskSdk/InputFunctions.ps1

```diff
@@ -15,14 +17,14 @@ function Get-Endpoint {
         [string]$Name,
```

Comment:
Get-Endpoint and Get-Input has been switched over to store the values securely and wipe the environment variable. Get-TaskVariable has not been switched over and should remain as is.